exercises/rna-transcription/example/RNA.ceylon

```diff
@@ -0,0 +1,13 @@
+class Error() {}
+
+String|Error transcription(String dna) {
+  return dna.fold("" of String|Error)((String|Error r,
+                                       Character    c) =>   if(is Error r)
+                                                          then(r)
```

Comment:
It is valid but I feel like it breaks the consistency with the parentheses on the if and else branches of the if statement. If you'd prefer them removed, I can do that.